src/Symfony/Bundle/FrameworkBundle/Resources/config/console.php

```diff
@@ -293,5 +294,14 @@
                 service('secrets.local_vault'),
             ])
             ->tag('console.command', ['command' => 'secrets:encrypt-from-local'])
+
+        ->set('console.command.mailer_debug', MailerDebugCommand::class)
+            ->tag('console.command', ['command' => 'debug:mailer:email:send'])
     ;
+
+    if (class_exists(MailerDebugCommand::class)) {
```

Comment:
Allow to remain BC if the symfony/mailer component is not installed.

Maybe there is a better way ?

Reply:
This won't play well with cases where the component code is available but the mailer is not configured. What you need is to define the service, and remove it again in FrameworkExtension based on the fact that the mailer is configured (and not just installed)